### PR TITLE
Fix openshift/client-go version

### LINKED ISSUE #7196: Cannot import kubevirt.io/client-go to a new project
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:
I've created a new go project and tried to import `kubevirt.io/client-go`:
```
go get kubevirt.io/client-go@v0.49.0
```

The command fails with the following error:
```
go: downloading kubevirt.io/client-go v0.49.0
go get: kubevirt.io/client-go@v0.33.0 updating to
	kubevirt.io/client-go@v0.49.0 requires
	github.com/openshift/client-go@v0.0.0: reading github.com/openshift/client-go/go.mod at revision v0.0.0: unknown revision v0.0.0
```

**What you expected to happen**:
I expected to be able to import the library.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:
`go version go1.16.9 linux/amd64`

### PR DESCRIPTION
**Pull request by fossedihelm**
**What this PR does / why we need it**:
`github.com/openshift/client-go  v0.0.0` is an unknown revision and makes it impossible to get `kubevirt/client-go` package, even if it is defined in the `replace` directive.

Signed-off-by: fossedihelm <ffossemo@redhat.com>
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #7196 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/kubevirt.io/client-go/go.mod [hunk 1]
```diff
@@ -14,7 +14,7 @@ require (
 	github.com/kubernetes-csi/external-snapshotter/v2 v2.1.1
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.2
-	github.com/openshift/client-go v0.0.0
+	github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
```
Comment (xpivarc) [inline staging/src/kubevirt.io/client-go/go.mod:17]: Can we remove the replace now?

Comment (fossedihelm) [reply staging/src/kubevirt.io/client-go/go.mod:17]: Sure

Comment (fossedihelm) [reply staging/src/kubevirt.io/client-go/go.mod:17]: Done

**Review by orelmisan — APPROVED** [review]
I've tested the changes using @fossedihelm fork:
`go get github.com/fossedihelm/client-go@test_mod`

Works for me.

**Comment by rmohr** [comment]
/approve 

:+1:

**Review by xpivarc — APPROVED** [review]

**Pull request merged**